assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -824,8 +824,9 @@ const baseSelectors = {
 		invariant( moduleSlug, 'slug is required.' );
 		const modules = select( STORE_NAME ).getModules();
 
-		if ( ! modules ) {
-			return null;
+		// Return `undefined` if modules haven't been loaded yet.
+		if ( modules === undefined ) {
+			return undefined;
```

Comment:
@danielgent could you, please, also update jsdoc for this selector to mentioned that it can return an undefined?

Reply:
`@return` still needs `undefined` added as a potential type.